csrankings-m.csv

```diff
@@ -1155,7 +1155,7 @@ Matthew T. Mason,Carnegie Mellon University,http://www.cs.cmu.edu/~mason,D_JpQnA
 Matthew Thazhuthaveetil,IISc Bangalore,http://www.csa.iisc.ac.in/~mjt,NOSCHOLARPAGE
 Matthew Thomas Mason,Carnegie Mellon University,http://www.cs.cmu.edu/~mason,D_JpQnAAAAAJ
 Matthew Toews,ETS Montreal,http://www.matthewtoews.com,EJaVEJIAAAAJ
-Matthew Turk,Univ. of California - Santa Barbara,http://www.cs.ucsb.edu/~mturk,KltleWgAAAAJ
+Matthew Turk, TTI-Chicago, https://home.ttic.edu/~mturk,KltleWgAAAAJ

```

Comment:
Please remove spurious spaces after commas and remove hyphen - institution name has no hyphen (in CSrankings).